src/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+
+function AddButton({ children }) {
+  return (
+    <button type="button" onClick={children}>
+      추가
+    </button>
+  );
+}
+
+function InputForm({ text, onClick }) {
+  return (
+    <div>
+      <form>
+        <input key="Form" type="text" placeholder="할 일을 입력해 주세요" />
+        <AddButton>{onClick}</AddButton>
+      </form>
+      <p>{text}</p>
+    </div>
+  );
+}
+
+function Page({ text, onClick }) {
+  return (
+    <InputForm
+      text={text}
+      onClick={onClick}
+    />
+  );
+}
+
+function ToDoList({ todo, onClick }) {
+  return (
+    <ol>
+      <li>{todo}</li>
+      <button type="button" onClick={onClick}>완료</button>
+    </ol>
+  );
+}
+
+function App() {
+  const [state, setState] = useState({
+    text: '할 일이 없어요!',
+    todo: '',
+  });
+
+  const { text, todo } = state;
+
+  function resultText() {
+    setState({
+      text: '',
+      todo: 'wow',
+    });
```

Comment:
```suggestion
    setState({
      ...state,
      text: '',
      todo: 'wow',
    });
```

새로 상태를 추가할 때 기존 상태를 이렇게 추가해주셔야해요.

Reply:
넵! 😃